content/tutorial/tutorial.md

```diff
@@ -1052,7 +1052,7 @@ Let's `map` over the `history` in the Game's `render` method:
 
 **[View the full code at this point](https://codepen.io/gaearon/pen/EmmGEa?editors=0010)**
 
-For each move in the tic-tac-toes's game's history, we create a list item `<li>` which contains a button `<button>`. The button has a `onClick` handler which calls a method called `this.jumpTo()`. We haven't implemented the `jumpTo()` method yet. For now, we should see a list of the moves that have occurred in the game and a warning that says:
+For each move in the tic-tac-toes's game's history, we create a list item `<li>` which contains a button `<button>`. The button has a `onClick` handler which calls a method called `this.jumpTo()`. We haven't implemented the `jumpTo()` method yet. For now, we should see a list of the moves that have occurred in the game and a warning in the browser dev console that says:
```

Comment:
I don't think "dev console" is a common way to say it. IMO either just "browser console" or "developer tools console" would be better.

Reply:
Also if you're following along using the CodePen, you will actually see the warning in the rendered output itself.

Reply:
Yes, see #1163

Reply:
That was precisely the problem as I described in #1163 ... I thought my page should look like the CodePen example and spent a really long time trying to find the “missing” warning. Perhaps it would be better to change CodePen ex to show console so it matches what people would see in their own dev environment?

Reply:
changed the wording to "developer tools console"